bugtool/cmd/root.go

```diff
@@ -0,0 +1,354 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/daemon/defaults"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// RootCmd is the top level command for the bugtool.
+var RootCmd = &cobra.Command{
+	Use:   "bugtool",
+	Short: "Cilium agent debugging tool",
+	Long:  "cilium-bugtool - capture system and node information for debugging a Cilium node",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("bugtool")
+		runTool()
+	},
+}
+
+const disclaimer = `DISCLAIMER
+This tool has copied information about your environment.
+If you are going to register a issue on GitHub, please
+only provide files from the archive you have reviewed
+for sensitive information. For more information on what
+is being collected, please see[0].
+
+Thanks.
+
+[0]: http://docs.cilium.io/en/stable/troubleshooting/#single-node-bugtool
+`
+
+var (
+	serve        bool
+	port         int
+	dumpPath     string
+	host         string
+	k8sNamespace string
+	k8sLabel     string
+	log          = logrus.New()
+)
+
+func init() {
+	RootCmd.Flags().BoolVar(&serve, "serve", false, "Start HTTP server to serve static files")
+	RootCmd.Flags().IntVarP(&port, "port", "p",
+		4444, "Port to use for the HTTP server, (default 4444)")
```

Comment:
I think this was split to meet a 80 char limit but line 60 doesn't. I'd vote to have it run long and be on one line (but go has something like a 120 char limit, if any).

Reply:
Can't remember why this was done. Not intentional.